Firestore/Source/API/FIRQuery+Internal.h

```diff
@@ -47,6 +48,12 @@ NS_ASSUME_NONNULL_BEGIN
 // TODO(orquery): This method will become public API. Change visibility and add documentation.
 - (FIRQuery *)queryWhereFilter:(FIRFilter *)filter;
 
+// TODO(COUNT): This method will become public API. Change visibility and add documentation.
```

Comment:
my obj-c knowledge is clearly letting me down. Is this a method?

Reply:
Copy/Paste error.